src/profile-logic/profile-data.js

```diff
@@ -1758,3 +1776,23 @@ export function getFriendlyStackTypeName(
       throw assertExhaustiveCheck(implementation);
   }
 }
+
+export function shouldDisplaySubcategoryInfoForCategory(
+  category: Category
+): boolean {
+  // The first subcategory of every category is the "Other" subcategory.
+  // For categories which only have the "Other" subcategory and no other
+  // subcategories, don't display any subcategory information.
+  return category.subcategories.length > 1;
+}
+
+export function getCategoryPairLabel(
+  categories: CategoryList,
+  categoryIndex: number,
+  subcategoryIndex: number
+) {
+  const category = categories[categoryIndex];
+  return shouldDisplaySubcategoryInfoForCategory(category)
+    ? `${category.name}: ${category.subcategories[subcategoryIndex]}`
+    : `${category.name}`;
+}
```

Comment:
proposal: if the subcategory is `Other`, do not display it.

When using the preview, I find that it doesn't bring much value to see for example `Layout: Other`, it could just be `Layout`. Then here we don't need to use `shouldDisplaySubcategoryInfoForCategory` but rather check the subcategory value.
(I understand we use `shouldDisplaySubcategoryInfoForCategory` for the sidebar in the other PR).

BTW I think my preference would have been to have `null` as value for the subcategory in this  case, instead of a `Other` category, but I assume you discussed this in the past and I don't want to make you do this again, while it's just a matter of preference.

Reply:
I agree, seeing "Other" in the tooltip is annoying so I'll do what you suggested.